doc/topics/windows/multi-minion.rst

```diff
@@ -0,0 +1,164 @@
+  - root_dir\\conf\\pki
+  - root_dir\\var\\log\\salt
+  - root_dir\\var\\run
+  - root_dir\\var\\cache\\salt\\minion\\extmods\\grains
+  - root_dir\\var\\cache\\salt\\minion\\proc
+
+.. code-block:: powershell
+
+    $RootDir = "<new root_dir location>"
+    $cache_dir = "$RootDir\var\cache\salt\minion"
+    New-Item -Path "$RootDir\conf" -Type Directory
+    New-Item -Path "$RootDir\conf\minion.d" -Type Directory
+    New-Item -Path "$RootDir\conf\pki" -Type Directory
+    New-Item -Path "$RootDir\var\log\salt" -Type Directory
+    New-Item -Path "$RootDir\var\run" -Type Directory
+    New-Item -Path "$cache_dir\extmods\grains" -Type Directory
+    New-Item -Path "$cache_dir\proc" -Type Directory
+
+4. Write minion config
+----------------------
+
+The minion will need is own config, separate from the system minion config. This
+config tells the minion where everything is as well as defines the master and
+minion id. Create a minion config file named ``minion`` in the conf directory.
+
+.. code-block:: powershell
+
+    New-Item -Path "$env:LocalAppData\Salt Project\Salt\conf\minion" -Type File
+
+Make sure the config file has at least the following contents:
+
+.. code-block:: yaml
+
+    master: <ip address, dns name, etc>
+    id: <minion id>
+
+    root_dir: <root_dir>
+    log_file: <root_dir>\val\log\salt\minion
+    utils_dirs:
+      - <root_dir>\var\cache\salt\minion\extmods
+    winrepo_dir: <root_dir>\srv\salt\win\repo
+    winrepo_dir_ng: <root_dir>\srv\salt\win\repo-ng
+
+    file_roots:
+      base:
+        - <root_dir>\srv\salt
+        - <root_dir>\srv\spm\salt
+
+    pillar_roots:
+      base:
+        - <root_dir>\srv\pillar
+        - <root_dir>\srv\spm\pillar
+
+    thorium_roots:
+      base:
+        - <root_dir>\srv\thorium
+
+Run the minion
+--------------
+
+Everything is now set up to run the minion. You can start the minion as you
+would normally, but you need to specify the full path to the config file you
+created above.
+
+.. code-block:: powershell
+
+    salt-minion.exe -c <root_dir>\conf
+
+Register the minion as a service
+--------------------------------
+
+You can also register the minion as a service, but you need to understand the
+implications of doing so.
+
+- You will need to have administrator privileges to register this minion service
+- You will need the password to the user account that will be running the minion
+- If the user password changes, you will have to update the service definition
+  to reflect the new password
+- The minion will run all the time under the user context, whether that user is
+  logged in or not
+- This requires great trust from the user as the minion will be able to perform
+  operations under the user's name without the user knowing, whether they are
+  logged in or not
+- If you decide to run the new minion under the Local System account, it might
+  as well just be a normal minion
+- The helper script does not support registering the 2nd minion as a service
```

Comment:
**Suggestions (phrasing, style, non-blocking):** Couple of wording suggestions to change things to the present tense and conform with style. I kept "will" in a few places where it felt right. FYI, Google likes you to end all list items with periods.

```
- You need to have administrator privileges to register this minion service.
- You need the password to the user account that will run the minion.
- If the user password changes, you will have to update the service definition
  to reflect the new password.
- The minion runs all the time under the user context, whether that user is
  logged in or not.
- This requires great trust from the user as the minion will be able to perform
  operations under the user's name without the user knowing, whether they are
  logged in or not.
- If you decide to run the new minion under the Local System account, it might
  as well just be a normal minion.
- The helper script does not support registering the second minion as a service.
```